src/resolvers/questionOption.ts

```diff
@@ -74,22 +74,5 @@ export const resolvers: Resolvers = {
         return await QuestionOption.findByQuestionOptionId('updateQuestion resolver', context, updatedQuestionOption.id);
       }
     },
-    removeQuestionOption: async (_, { questionOptionId }, context: MyContext): Promise<QuestionOption> => {
```

Comment:
I was also thinking about situations like this. There are a few cases where we exposed individual `removeThing` resolvers, but the front end sends the list of its associations when the parent object gets created/updated. 

As part of the ticket I mentioned above, we should probably figure out which approaches we are using for each scenario and delete any unused ones